pandas/core/strings.py

```diff
@@ -1161,6 +1161,24 @@ def rfind(self, sub, start=0, end=None):
         result = str_find(self.series, sub, start=start, end=end, side='right')
         return self._wrap_result(result)
 
+    def normalize(self, form):
+        """Return the Unicode normal form for the strings in the Series/Index.
+        Equivalent to standard :func:`unicodedata.normalize`.
+
+        Parameters
+        ----------
+        form : {'NFC', 'NFKC', 'NFD', 'NFKD'}
+            Unicode form
```

Comment:
Maybe add something like: "For more information on the forms, see the :func:`unicodedata.normalize` docs" ?